src/js/components/add-journal-entry/journal-entry.js

```diff
@@ -69,6 +100,12 @@ class JournalEntry extends MaharaBaseComponent {
         //console.log("new tags", tags);
         this.tags = tags; // parent component accesses it this way
     }
+
+    changeJournal() {
+        let targetBlogId = parseInt(this.refs.journalSelect.el.select2('data')[0].id, 10);
```

Comment:
Oopps ... Is there a better way to get the ID? For example store `selectedBlogId` in the state, and set this value upon `onChange` of the select